spring-integration-file/src/main/java/org/springframework/integration/file/dsl/FileTransformers.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.file.dsl;
+
+import org.springframework.integration.file.transformer.FileToByteArrayTransformer;
+import org.springframework.integration.file.transformer.FileToStringTransformer;
+
+/**
+ * Factory methods for file transformers.
+ *
+ * @author Artem Bilan
+ * @author Gary Russell
+ * @since 5.2
+ *
+ */
+public final class FileTransformers {
+
+	private FileTransformers() {
+		super();
+	}
+
+	public static FileToStringTransformer fileToString() {
```

Comment:
Can't we have these simple factory methods just directly in the `Files` class?
And make that `final`, too 😉